### PR TITLE
Make the Readme File More Valuable

### LINKED ISSUE #3: Our Second Issuel

### LINKED ISSUE #4: Our Third Issue

### PR DESCRIPTION
**Pull request by gulecas**
Created a much better Readme layout as well as an example readme layout so that we can better inform people
about what our repository is all about.

Fixes #3

### CODE REVIEW

File: README.md
Comment (github-eg-g) [inline README.md:1]: Looks good to me.

File: ReadmeStructure.md
Comment (github-eg-g) [inline ReadmeStructure.md:1]: I think you should add spaces between some headings.

**Review by github-eg-g — CHANGES_REQUESTED** [review]
Address what I said.

File: ReadmeStructure.md
Comment (gulecas) [reply ReadmeStructure.md:1]: This is the format.

**Review by gulecas — COMMENTED** [review]
I think I'm done.

**Review by gulecas — COMMENTED** [review]
I think, I'm done this time

**Review by github-eg-g — APPROVED** [review]
Ok, this looks fine now.

**Pull request merged**